rgb_throttle:
  ros__parameters:
    input_topic: "/rgb"
    output_topic: "/rgb/throttled"
    timer_frequency: 10.0  # Hz

    resize_enabled: false
    resize_width: 300
    resize_height: 200

    input_qos:
      reliability: "best_effort"
      durability: "volatile"
      history: "keep_last"
      depth: 10

    output_qos:
      reliability: "best_effort"
      durability: "volatile"
      history: "keep_last"
      depth: 5

depth_throttle:
  ros__parameters:
    input_topic: "/depth"
    output_topic: "/depth/throttled"
    timer_frequency: 10.0  # Hz

    resize_enabled: false
    resize_width: 300
    resize_height: 200

    input_qos:
      reliability: "best_effort"
      durability: "volatile"
      history: "keep_last"
      depth: 10

    output_qos:
      reliability: "best_effort"
      durability: "volatile"
      history: "keep_last"
      depth: 5
